packages/ui/src/TapToEdit.tsx

```diff
@@ -110,6 +110,23 @@ export const TapToEdit = ({
       } else if (fieldProps?.type === "multiselect") {
         // ???
         displayValue = value.join(", ");
+      } else if (fieldProps?.type === "address") {
+        const city = value?.city
+          ? value?.state || value.zipcode
+            ? `${value.city}, `
+            : `${value.city}`
+          : "";
+        const state = value?.state ? (value?.zipcode ? `${value.state} ` : `${value.state}`) : "";
+        const zip = value?.zipcode;
+
+        const addressLineOne = value?.address1 ? value?.address1 : "";
```

Comment:
I believe this can be `const addressLineOne = value?.address1 ?? "";` or the older (looser, any fasley check) `const addressLineOne = value?.address1 || "";`. The ternary is a little confusing.

Reply:
done! :)